### PR TITLE
Chap 4 :recycle: Renaming from code review

### PR DESCRIPTION
**Pull request by ablqk**
Renaming a few things in Chapter 4 as per Egon's review

### CODE REVIEW

File: chapter-04/final/main.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func readCorpus(path string) ([]string, error) {
 }
 
 func main() {
-	fmt.Println("Welcome to Gordle!")
+	fmt.Println("Welcome to Game!")
```
Comment (floppyzedolfin) [inline chapter-04/final/main.go:27]:
```suggestion
	fmt.Println("Welcome to Gordle!")
```

Comment (doniacld) [reply chapter-04/final/main.go]: Fixed.

File: chapter-04/1_1_basic_main/gordle/game.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	"fmt"
+)
+
+// Game holds all the information we need to play a game of gordle.
+type Game struct{}
+
+// New returns a Game variable, which can be used to Play!
+func New() *Game {
+	g := &Game{}
+	fmt.Println("Welcome to Gordle!")
```
Comment (ablqk) [inline chapter-04/1_1_basic_main/gordle/game.go:13]: This should be part of the Play() method, don't you think?

File: chapter-04/1_4_check_for_victory/gordle/game.go [hunk 3]
```diff
@@ -7,34 +7,35 @@ import (
...
 	// break condition: we've reached the maximum number of attempts
 	for currentAttempt := 1; currentAttempt <= g.maxAttempts; currentAttempt++ {
 		// ask for a valid word
-		attempt := g.ask()
+		guess := g.ask()
 
-		if string(attempt) == string(g.solution) {
-			fmt.Printf("🎉 You won! You found in %d attempt(s)! The word was: %s.\n", currentAttempt, string(g.solution))
+		if string(guess) == string(g.solution) {
+			fmt.Printf("🎉 You won! You found in %d guess(s)! The word was: %s.\n", currentAttempt, string(g.solution))
```
Comment (floppyzedolfin) [inline chapter-04/1_4_check_for_victory/gordle/game.go:38]:
```suggestion
			fmt.Printf("🎉 You won! You found in %d guess(es)! The word was: %s.\n", currentAttempt, string(g.solution))
```

File: chapter-04/2_feedback/gordle/game.go [hunk 4]
```diff
@@ -7,42 +7,43 @@ import (
...
 		// evaluate it
-		fb := g.solutionChecker.evaluate(attempt)
+		fb := g.solutionChecker.evaluate(guess)
 
 		// print the feedback
 		fmt.Println(fb.String())
 
-		if string(attempt) == string(g.solution) {
+		if string(guess) == string(g.solution) {
 			fmt.Printf("🎉 You won! You found in %d attempt(s)! The word was: %s.\n", currentAttempt, string(g.solution))
```
Comment (floppyzedolfin) [inline chapter-04/2_feedback/gordle/game.go:47]: guess(es) ?

File: chapter-04/3_corpus/gordle/game.go [hunk 5]
```diff
@@ -8,45 +8,46 @@ import (
...
 		// check it
-		fb := g.solutionChecker.check(attempt)
+		fb := g.solutionChecker.check(guess)
 
 		// print the feedback
 		fmt.Println(fb.String())
 
-		if string(attempt) == string(g.solution) {
+		if string(guess) == string(g.solution) {
 			fmt.Printf("🎉 You won! You found in %d attempt(s)! The word was: %s.\n", currentAttempt, string(g.solution))
```
Comment (floppyzedolfin) [inline chapter-04/3_corpus/gordle/game.go:51]: guess(es)

**Review by floppyzedolfin — APPROVED** [review]

**Pull request merged**